br/pkg/task/stream.go

```diff
@@ -740,24 +765,29 @@ func initFullBackupTables(ctx context.Context, fullBackupStorage string, cfg *St
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	tables := make(map[string]*metautil.Table)
+
+	tables := make(map[int64]*metautil.Table)
 	for _, db := range databases {
 		dbName := db.Info.Name.O
 		if name, ok := utils.GetSysDBName(db.Info.Name); utils.IsSysDB(name) && ok {
 			dbName = name
 		}
+
+		if !cfg.TableFilter.MatchSchema(dbName) {
+			continue
+		}
+
 		for _, table := range db.Tables {
 			if !cfg.TableFilter.MatchTable(dbName, table.Info.Name.O) {
 				continue
 			}
-			tables[utils.UniqueID(dbName, table.Info.Name.String())] = table
```

Comment:
UniqueID can be removed since it's not used

Reply:
> UniqueID can be removed since it's not used

it has replace for `tables[table.Info.ID] = table` in this pr.

Reply:
yes. so I say, you can remove the function `UniqueID` since it's no longer being used and more.